app/helpers/browser_helper.rb

```diff
@@ -32,7 +29,7 @@ def unsupported_browser?
   # or mobile detection
   def browser_specific_classes
     [].tap do |classes|
-      classes << "-browser-chrome" if browser.chrome?
+      classes << "-browser-chrome" if browser.chrome? || browser.chromium_based?
```

Comment:
Maybe can remove `browser.chrome?` and below `browser.edge?`, but will also break syntax meaning of class name *-browser-chrome*, so the first choose is only add 'or' condition here.